### PR TITLE
[CHK-1017] Unused code refactoring

### PR DESCRIPTION
**Pull request by MarcoAGreco**
#### List of Changes
- Remove unused code after refactoring

#### Motivation and Context
- Improve code quality

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/utils/api/apiService.ts [hunk 1]
```diff
@@ -186,10 +132,6 @@ export function setCart(item: Cart) {
   sessionStorage.setItem(SessionItems.cart, JSON.stringify(item));
 }
 
-export function setWallet(item: Wallet) {
-  sessionStorage.setItem(SessionItems.wallet, JSON.stringify(item));
```
Comment (Fernando-Granato) [inline src/utils/api/apiService.ts:190]: You have to merge conflicts

File: src/utils/api/helper.ts [hunk 2]
```diff
@@ -436,7 +334,7 @@ export const activePaymentTask = (
     )
   );
 
-export const getTransactionData = async ({
+const getTransactionData = async ({
```
Comment (infantesimone) [inline src/utils/api/helper.ts:337]: why remove export?

Comment (MarcoAGreco) [reply src/utils/api/helper.ts:337]: This const is not used outside, so I thought  it was unnecessary

File: src/utils/api/helper.ts [hunk 3]
```diff
@@ -1061,169 +525,17 @@ export const proceedToPayment = async (
...
-export const cancelPayment = async (
-  onError: (e: string) => void,
-  onResponse: () => void
-) => {
-  const checkData = getSessionItem(SessionItems.checkData) as
-    | PaymentCheckData
-    | undefined;
+// TODO Invoke ecommerce api to perform cancel payment
+export const cancelPayment = async (onResponse: () => void) => {
   // Payment action DELETE
```
Comment (Fernando-Granato) [inline src/utils/api/helper.ts:530]: TODO CHK-1007 !

**Review by Fernando-Granato — APPROVED** [review]
Well done Gianluca  🚀 !!

**Pull request merged**